src/utils/__tests__/dockerDirManager.spec.ts

```diff
@@ -400,34 +402,35 @@ describe('DockerDirManager', () => {
   });
 
   describe('credHelperWorking', () => {
-    let fakeProcess: childProcess.ChildProcess;
-    let spawnMock: jest.SpiedFunction<typeof childProcess.spawn>;
-
-    beforeAll(() => {
-      spawnMock = jest.spyOn(childProcess, 'spawn')
-        .mockImplementation(() => {
-          fakeProcess = new childProcess.ChildProcess();
-
-          return fakeProcess;
-        });
-    });
-
-    afterAll(() => {
+    let spawnMock: jest.SpiedFunction<typeof childProcess.spawnFile>;
+    const commonCredHelperExpectations = (command: any, args: any, options: any) => {
```

Comment:
```suggestion
    const commonCredHelperExpectations: (...args: Parameters<typeof childProcess.spawnFile>) => void = (command, args, options) => {
```
The types here should be constant?

This means we can drop the cast in the `list` check.

Reply:
Yes, done.